docs/adding games.md

```diff
@@ -72,7 +75,7 @@ start of the game from anywhere
 * Regions are simply containers for locations that share similar access rules. They do not have to map to 
 concrete, physical areas within your game and can be more abstract like tech trees or a questline.
 
-The base World class can be found in [AutoWorld](/worlds/AutoWorld.py). Methods available for your world to call during
-generation can be found in [BaseClasses](/BaseClasses.py) and [Fill](/Fill.py). Some examples and documentation 
-regarding the API can be found in the [world api doc](/docs/world%20api.md).
-Before publishing, make sure to also check out [world maintainer.md](/docs/world%20maintainer.md).
+The base World class can be found in [AutoWorld](autoworld). Methods available for your world to call during
+generation can be found in [BaseClasses](baseclasses) and [Fill](fill). Some examples and documentation 
```

Comment:
Not sure how many other instances of this are out there, but If people view this as markdown, those links will be broken.

Reply:
i couldn't think of any good alternative for this. the docs it's linking to aren't markdown, but i'd much prefer to link to docs than to the source code when possible, since it's pretty jarring for the docs to suddenly go to GitHub for something that already exists there.

Reply:
I'd suggest having a map/dict and do string replace to generate "correct" links for sphinx rather than having the wrong links in the plain markdown maybe?